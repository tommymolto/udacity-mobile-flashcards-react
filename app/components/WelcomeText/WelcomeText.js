import React from 'react';
import { View } from 'react-native';
import styled from 'styled-components/native';

const Header = styled.Text`
  font-size: 24;
  text-align: center;
  padding-top: 10;
  padding-horizontal: 15;
`;

const SubHeader = Header.extend`
  font-size: 18;
  padding-bottom: 10;
`;

const WelcomeText = () => (
  <View>
    <Header>Welcome to Mobile Flashcards!</Header>
    <SubHeader>What do you want to learn today?</SubHeader>
  </View>
);

export default WelcomeText;
