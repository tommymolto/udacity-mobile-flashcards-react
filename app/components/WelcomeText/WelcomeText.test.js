/* global describe, it, expect */

import React from 'react';
import renderer from 'react-test-renderer';
import WelcomeText from './WelcomeText';

describe('WelcomeText', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<WelcomeText />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
