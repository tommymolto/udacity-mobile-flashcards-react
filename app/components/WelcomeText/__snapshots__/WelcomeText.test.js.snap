// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomeText renders correctly 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontSize": 24,
          "paddingHorizontal": 15,
          "paddingTop": 10,
          "textAlign": "center",
        },
        undefined,
      ]
    }
  >
    Welcome to Mobile Flashcards!
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontSize": 18,
          "paddingBottom": 10,
          "paddingHorizontal": 15,
          "paddingTop": 10,
          "textAlign": "center",
        },
        undefined,
      ]
    }
  >
    What do you want to learn today?
  </Text>
</View>
`;
